backend/Database/product.js

```diff
@@ -0,0 +1,53 @@
+const { Decimal128 } = require("bson");
```

Comment:
Decimal128 hasn't been used anywhere. I believe this is not necessary. The schema type 'Number' accepts floating point values as well.

Reply:
done